src/providers/k8s/kubeClient.ts

```diff
@@ -330,7 +333,6 @@ export class KubeClient extends Client {
     if (unique) {
       const args = ["cp", localFilePath, `${identifier}:${podFilePath}`];
       if (container) args.push("-c", container);
-      const result = await this.runCommand(args, undefined, true);
```

Comment:
Why we are removing this line? we should do
```js
await this.runCommand(args, undefined, true);
```

Reply:
Erroneously probably during some merge. Fixed it. Apologies about that

Reply:
Awesome, don't need to apology :)